src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -49,15 +49,16 @@
     String escapedTableName;
     List<DBColumn> columns;
     int totalColumns;
-    int totalRows = 2; // default row count set to 2
+    static int totalRows = 3; // default row count set to 3
     DBSchema schema;
+    ArrayList<ArrayList<Object>> rows;
```

Comment:
rows is all rows we need to verify our backend data. verification is done row by row.